pkg/apis/config/feature_flags.go

```diff
@@ -62,6 +62,8 @@ const (
 	DefaultSendCloudEventsForRuns = false
 	// DefaultEmbeddedStatus is the default value for "embedded-status".
 	DefaultEmbeddedStatus = FullEmbeddedStatus
+	// DefaultEnableSpire is the default value for "enable-spire".
+	DefaultEnableSpire = false
```

Comment:
NIT: The flag is not in the config map YAML under `config/` and not in the docs, which makes sense since the feature is not available yet. Perhaps we could have a comment here saying that the feature is not yet available?

Reply:
This comment would be invalid in an immediate next PR. Will skip for now. Let me know if you feel strongly about it and I can add it.